hack/aks

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
```

Comment:
What about something with "terraform" in the name, since there is a dependency on it

What about `tf-aks` or `aks-tf` to keep it short?

Reply:
will do.